src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -220,18 +219,17 @@ internal static void FreeSslContext(SafeSslHandle context)
         #endregion
 
         #region private methods
-        private static IntPtr GetSslMethod(bool isServer, long options)
+        private static IntPtr GetSslMethod(SslProtocols protocols)
```

Comment:
> We deliberately didn't use SslProtocols because that introduces a dependency on System.Net.Security. 

The SslProtocols enum is defined in System.Net.Primitives (https://github.com/dotnet/corefx/blob/c02d33b18398199f6acc17d375dab154e9a1df66/src/System.Net.Primitives/ref/System.Net.Primitives.cs#L413)